src/libs/ui/src/molecules/AlreadyHaveAnAccountCard/AlreadyHaveAnAccountCard.styles.scss

```diff
@@ -0,0 +1,26 @@
+.twocards-card-two{

```

Comment:
Wrap all your component css class in one container

Reply:
I have Wrapped in one container